apps/sms/test/marionette/composer_test.js

```diff
@@ -257,38 +258,73 @@ marionette('Messages Composer', function() {
   });
 
 
-  suite('Invalid recipients', function() {
+  suite('Recipients list', function() {
     var newMessage;
+    var storage;
+    var contact = {
+      name: ['Existing Contact'],
+      givenName: ['Existing'],
+      familyName: ['Contact'],
+      tel: [{
+        value: '5551234567',
+        type: 'Mobile'
+      }]
+    };
 
     setup(function() {
-      messagesApp.launch();
+      var MOCKS = [
```

Comment:
nit: move ```MOCKS``` in the parent scope, where the rest of "const" variables (```newMessage storage contact```) are defined